internal/profile/db.sql

```diff
@@ -1,5 +1,21 @@
-
-CREATE TABLE profiles (
+CREATE TABLE IF NOT EXISTS profiles (
     user_id varchar(255) PRIMARY KEY,
     profile json NOT NULL
 );
+
+ALTER TABLE profiles ADD COLUMN version BIGINT default(extract(epoch from now()) * 1000);
+
+CREATE OR REPLACE FUNCTION update_version()
```

Comment:
you can have an updated_at field by just using DEFAULT now() or current_timestamp (http://www.postgresqltutorial.com/postgresql-current_timestamp/), does this not work because version is not a timestamp? because it may be easier to just store a timestamp and generate the version on the fly

Reply:
yes at the beginning it was a timestamp, and i was generating the "version" on the fly, but i thought it might me annoying to debug an issue and query the DB if we have the representation of the data and not the real data, unless we use microseconds resolutions for the ids, otherwise we will be loosing data.

Reply:
I think you can specify precision on current timestamp, this way will could map any given ms into a timempstamp object

Reply:
now I think about it, what about this:

- let's add a created_at and updated_at column for debugging purposes with the current timestamp.
- let's use a normal serie for the version

Reply:
i like this last option, i didm't like this duality in the version/update field. I will do that.